docs/tsconfig.node.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "skipLibCheck": true,
+    "module": "ESNext",
+    "moduleResolution": "bundler",
+    "allowSyntheticDefaultImports": true
+  },
+  "include": ["vite.config.ts"]
+}
```

Comment:
I don't understand why we need this file? We can include vite.config.ts in the regular project

Reply:
This stackoverflow may have the best answer as to why tsconfig.node.json is needed. In short, this file is the tsconfig used by vite itself. On the other hand, react in the src folder refers to tsconfig.json

https://stackoverflow.com/questions/72027949/why-does-vite-create-two-typescript-config-files-tsconfig-json-and-tsconfig-nod

Reply:
yea, I just noticed i have this file too in one of my recent vite projects